### PR TITLE
Base URL (Secure) config fields

### PR DESCRIPTION
**Pull request by ccondra**
## Purpose of this pull request
The purpose of this PR is to add field descriptions for new fields added to the General > Web > Base URL (Secure) configuration, update the screenshot, add missing field to instructions in related topic.

## Affected documentation pages

[Web](https://docs.magento.com/m2/ee/user_guide/configuration/general/web.html)
Added new fields to Base URLs (Secure) configuration, and updated screenshot:

- 	Enable HTTP Strict Transport Security (HSTS)
- 	Upgrade Insecure Requests
- 	Offloader header

[Store URLs](https://docs.magento.com/m2/ee/user_guide/stores/store-urls.html)
Added Offloader header field to instructions in [Store URLs](https://docs.magento.com/m2/ee/user_guide/stores/store-urls.html)

## Affected Magento editions

- [ x ] Open Source
- [ x ] Commerce
- [ x ] B2B

### CODE REVIEW

File: src/stores/store-urls.md [hunk 1]
```diff
@@ -88,7 +88,9 @@ If your domain has a valid security certificate, you can configure the URLs of b
 
     - **Upgrade Insecure Requests** — To upgrade any requests for standard unsecured HTTP pages to secure HTTPS, set to `Yes`.
 
-1. When complete, click <span class="btn">Save Config</span>.
+1.  Most Magento installations use the default  `X-Forward-Proto` **Offloader Header** to identify the protocol as either `HTTP` or `HTTPS`. If your server configuration uses a different `offloader_header`, enter it here.
```
Comment (hickskenh) [inline src/stores/store-urls.md]: extra space after "default"

File: src/stores/store-urls.md [hunk 2]
```diff
@@ -13,14 +13,14 @@ Do not change the Admin URL from the default Base URL configuration. To change t
...
+Magento can be configured to deliver all pages over `HTTPS` by default. If your store has been running up until now with standard protocol, you can improve security by enabling [HTTP Strict Transport Security][2] (HSTS). and by upgrading any insecure page requests that are received by the store. HSTS is an opt-in protocol that prevents browsers from rendering standard `HTTP` pages that are transmitted with unsecure protocol for the specified domain. Because search engines might have already indexed each page of your store with standard `HTTP` URLs, you can configure Magento to automatically upgrade any unsecure page requests to `HTTPS`, so you don't lose any traffic. When Magento is configured to use secure URLs for both the store and Admin, two additional fields appear that allow you to enable `HSTS`.
```
Comment (hickskenh) [inline src/stores/store-urls.md]: remove "." between (HSTS). and by

on first instance of http, maybe add unsecure.  and on first instance of https, maybe add secure

File: src/configuration/general/web.md [hunk 3]
```diff
@@ -47,7 +47,10 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Gene
...
+|Use Secure URLs in Admin|Global|If your domain has a security certificate, you can choose to run the store Admin, with or without SSL encryption. Options: <br />**Yes** - Admin URLs begin with `https` to indicate that the page is delivered with encrypted, secure protocol.<br />**No** - Admin URLs begin with `http` to indicate that the page is delivered without secure protocol.<br /> When secure URLs are enabled for both the store and Admin, two additional fields appear to enable and configure `HSTS`.|
+|Enable HTTP Strict Transport Security (HSTS)|Store View|When enabled, [`HSTS`](1) provides a measure of security against "man in the middle" attacks, and prevents users from overriding the "invalid certificate" message. Options: Yes / No|
+|Upgrade Insecure Requests|Store View| When enabled, coverts insecure (`HTTP`) requests received from the browser to the secure (`HTTPS`) protocol. Options: Yes / No|
```
[the remark this answers is not in the document]
Comment (ccondra) [reply src/configuration/general/web.md]: ha! you're right!

File: src/stores/store-urls.md (hunk 1 above)
Comment (ccondra) [reply src/stores/store-urls.md]: fixed

**Comment by ccondra** [comment]
running tests

**Comment by ccondra** [comment]
running tests

**Comment by ccondra** [comment]
running tests

**Comment by ccondra** [comment]
running tests

**Comment by ccondra** [comment]
running tests

**Pull request merged**